### ISSUE TITLE
Bunthen Plainswind in Moonglade missing FP to Thunderbluff.

### ISSUE THREAD

**Issue by regneq** [mangosArchives/databaseZero_Rel19#74]
Clicked on the horde druid Flightmaster and there was no option to fly back to Thunderbluff.

**Comment by Ghurok** [comment]
Please provide more information.
1. You need to be a druid to be able to see the gossip option and use the flight path AND
2. You need to be horde.

I made a new horde druid character, teleported to Bunthen Plainswind with a GM command and was able to click on the gossip option and get on a flight to Thunderbluff.

Screenshot: http://puu.sh/3JpNL.jpg

Closing this issue for now.

**Comment by Ghurok** [comment]
This issue was tested by another developer and it did not work for him. Reopening the issue.

Going to investigate it further later.

**Comment by danielsreichenbach** [comment]
I have done a fresh installation with current mangos, scripts, and lastest revision of the database.

Talking to Bunthen Plainswind will show the appropriate dialogue, and let a druid fly to Thunderbluff.

Thus I am closing the issue for now.

**Comment by regneq** [comment]
yeah   at level 14 druid and went to pick up the cure poison quest in moonglade and went to see if this Flightmaster will let me fly back to Thunderbluff.  And once again, offer no flight back.  This is definately still broken.

**Comment by regneq** [comment]
here is a screenshot.
![wowscrnshot_080413_212918](https://f.cloud.github.com/assets/3778417/908248/a930a48a-fd88-11e2-8b55-fe3ffb476f1e.jpg)

**Comment by danielsreichenbach** [comment]
That is indeed interesting. So far none of us have been able to reproduce the issue on local testing installations.

The character you leveled was leveled on a normal account without any special user rights, no character modifications by someone with game master rights?

**Comment by regneq** [comment]
yep.  just a plain normal account only used by me with no special user
rights.
